pkg/rtc/dynacastquality.go

```diff
@@ -13,9 +13,8 @@ const (
 )
 
 type DynacastQualityParams struct {
-	MimeType           string
-	DynacastPauseDelay time.Duration
-	Logger             logger.Logger
+	MimeType string
+	Logger   logger.Logger
 }
```

Comment:
This and the above file, unrelated clean up. was trying to check quality and wanted to turn off dynacast. Mistakenly increased `DynacastPauseDelay` thinking that the initial dynacast quality reduction would be delayed by that much (that config is for debouncing quality reduction). But, realised what I was looking for was a constant. So, cleaned up the unused bit.